src/Kunstmaan/AdminBundle/Twig/MultiDomainAdminTwigExtension.php

```diff
@@ -0,0 +1,102 @@
+<?php
```

Comment:
I think the name should stay this way. It's an extension for the AdminBundle with functionality for the multidomain.